### PR TITLE
Fixed typos in Readme

### PR DESCRIPTION
**Pull request by G7-TheKing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,19 +40,19 @@
 
 We love the default GitHub profiles and we want to enhance them:
 
-* The GitHub profiles aren't clearly showing all repos you have contributed to since you joined
+* The GitHub profiles clearly aren't showing all the repos you have contributed to since you joined
```
Comment (AurelienLourot) [inline README.md]: 1. aren't clearly showing them = they show them, but not in a clear manner
2. clearly aren't showing them = they obviously don't show them at all

I do mean number 1 :)

Comment (AurelienLourot) [inline README.md]: 1. aren't clearly showing them = they show them, but not in a clear manner
2. clearly aren't showing them = they obviously don't show them at all

I do mean number 1 :)

File: README.md [hunk 2]
```diff
@@ -40,19 +40,19 @@
...
   GitHub. We are showing them **all**, even those you don't own and those owned by organizations
   you're not in.<sup>[1](#footnote1)</sup>
-* The GitHub profiles are listing all the repos you own but they sort them only by age of the
-  latest commit. We prefer to **sort repos** by a combination of how much you
-  have contributed to them, their size, how popular they are, etc. For each user we want to see
-  first the latest greatest repos they have most contributed to.
+* The GitHub profiles currently list all the repos you own which are sorted only by age of the
```
Comment (AurelienLourot) [inline README.md]: here also you're changing the meaning and it doesn't mean what we had in mind anymore.

File: README.md [hunk 3]
```diff
@@ -40,19 +40,19 @@
...
+  latest commit. We prefer to **sort repos** by a combination of how much you 
+  have contributed to them, their size, how popular they are, etc. For each user, we want to see
+  first the latest repos they have most contributed to the most.
```
Comment (AurelienLourot) [inline README.md]: Now you have "most" twice in the sentence...

File: README.md [hunk 4]
```diff
@@ -40,19 +40,19 @@
...
 * On GitHub only repos earn stars. We push it one step further by having **users earn stars**:
   You earn stars when you contribute to a repo.
   We add all these earned stars and show how many you've earned in total.
 * The GitHub profiles don't clearly show how big your contribution to a repo was, when you don't own
   it. Maybe you wrote 5%. Maybe 90%. We **make it clear**.
-* GitHub detects programming languages. We want to also know about
+* GitHub detects programming languages. We also want to know about
```
Comment (AurelienLourot) [inline README.md]: This one I agree :)

File: README.md (hunk 1 above)
Comment (AurelienLourot) [inline README.md]: 1. aren't clearly showing them = showing them, but not in a clear way
2. clearly aren't showing them = not showing them at all

We do mean 1 :)

"the" -> OK

Comment (AurelienLourot) [inline README.md]: 1. aren't clearly showing them = showing them, but not in a clear way
2. clearly aren't showing them = not showing them at all

We do mean 1 :)

"the" -> OK

**Comment by AurelienLourot** [comment]
(sorry for all the duplicated messages, GitHub was having [technical issues](https://twitter.com/githubstatus))

File: README.md (hunk 2 above)
Comment (G7-TheKing) [reply README.md]: I understand, by all means, revert this change

**Pull request merged**